api/src/main/java/com/avast/metrics/TimerPairImpl.java

```diff
@@ -54,4 +54,14 @@ public TimerPairImpl(Timer successTimer, Timer failureTimer) {
             throw ex;
         }
     }
+
+    @Override
+    public Timer getSuccessTimer() {
```

Comment:
Are you aware that this means that anyone can take out just one of these timers and mutate them (measure something with them) which means that their values will be incorrect (I would expect that both these timers always measure time together and it's not possible to use them separately). I know it's quite stupid to do this but that's why I don't like to put getter everywhere and more so when the thing is mutable.